S/seqgen/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "SeqGen"
```

Comment:
This name must match _exactly_ the directory name, including the case. I'd suggest renaming the directory.

Reply:
Thanks for the tips!